requirements.txt

```diff
@@ -1,5 +1,6 @@
 # List of requirements
-# Run 
-sympy>=1.1.1
-numpy>=1.13.1
+# Run
+python == 3.9.18
```

Comment:
You cannot add python version to requirements file. The file is for packages.

Reply:
Yes my bad ;-)